ius-6-x86_64.cfg

```diff
@@ -0,0 +1,103 @@
+config_opts['root'] = 'ius-6-x86_64'
+config_opts['target_arch'] = 'x86_64'
+config_opts['legal_host_arches'] = ('x86_64',)
+config_opts['chroot_setup_cmd'] = 'install @buildsys-build'
+config_opts['dist'] = 'el6'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '6'
+
+config_opts['yum.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+mdpolicy=group:primary
+best=1
+
+# repos
+[base]
+name=BaseOS
+mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=os
+failovermethod=priority
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-6
+gpgcheck=1
+skip_if_unavailable=False
+
+[updates]
+name=updates
+enabled=1
+mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=updates
+failovermethod=priority
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-6
+gpgcheck=1
+skip_if_unavailable=False
+
+[epel]
+name=epel
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=epel-6&arch=x86_64
+failovermethod=priority
+gpgkey=file:///usr/share/distribution-gpg-keys/epel/RPM-GPG-KEY-EPEL-6
+gpgcheck=1
+skip_if_unavailable=False
+
+[extras]
+name=extras
+mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=extras
+failovermethod=priority
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-6
+gpgcheck=1
+skip_if_unavailable=False
+
+[sclo]
+name=sclo
+baseurl=http://mirror.centos.org/centos/6/sclo/x86_64/sclo/
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-SIG-SCLo
+gpgcheck=1
+includepkgs=devtoolset*
+skip_if_unavailable=False
+
+[sclo-rh]
+name=sclo-rh
+baseurl=http://mirror.centos.org/centos/6/sclo/x86_64/rh/
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-SIG-SCLo
+gpgcheck=1
+includepkgs=devtoolset*
+skip_if_unavailable=False
+
+[testing]
+name=epel-testing
+enabled=0
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=testing-epel6&arch=x86_64
+failovermethod=priority
+skip_if_unavailable=False
+
+[local]
+name=local
+baseurl=https://kojipkgs.fedoraproject.org/repos/epel6-build/latest/x86_64/
+cost=2000
+enabled=0
+skip_if_unavailable=False
+
+[epel-debuginfo]
+name=epel-debug
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=epel-debug-6&arch=x86_64
+failovermethod=priority
+enabled=0
+skip_if_unavailable=False
+
+[ius]
+name=ius
+#baseurl=https://dl.iuscommunity.org/pub/ius/stable/CentOS/6/$basearch
+mirrorlist=https://mirrors.iuscommunity.org/mirrorlist?repo=ius-centos6&arch=$basearch&protocol=http
+failovermethod=priority
+enabled=1
+gpgcheck=1
+gpgkey=https://dl.iuscommunity.org/pub/ius/IUS-COMMUNITY-GPG-KEY
```

Comment:
This is the old config that points to the mirrorlist API server.  Can you update it to point directly to the CDN?  It should match the ius section from [ius-6.repo](https://github.com/iusrepo/ius-release/blob/master/ius-6.repo).

Just like we have epel-testing, it would also be useful to add ius-testing from [ius-testing-6.repo](https://github.com/iusrepo/ius-release/blob/master/ius-testing-6.repo).